manifests/device-plugin.yml

```diff
@@ -34,11 +33,11 @@ spec:
       # be rescheduled after a failure.
       # See https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
       priorityClassName: "system-node-critical"
-      # In case machine deploy MPS device plugin and change compute mode to
+      # Reset compute mode to DEFAULT
       initContainers:
       - name: set-compute-mode
         image: nvidia/cuda:latest
-        command: ['nvidia-smi', '-c', 'EXCLUSIVE_PROCESS']
+        command: ['nvidia-smi', '-c', 'DEFAULT']
```

Comment:
Did you meet problems using `DEFAULT`? This is not recommendation pattern because GPU should be fully controlled by MPS.